docs/articles/keyboard-shortcuts.md

```diff
@@ -11,6 +11,7 @@ Files comes with built in keyboard shortcuts to make naviagting the app easier.
 | `Ctrl` + `T` | Create new tab |
 | `Ctrl` + `W` | Close tab |
 | `Ctrl` + `F4` | Close tab |
+| `Ctrl` + `Shift` + `T` | Reopen closed tab |
```

Comment:
Can we make it clearer that this will not just reopen some closed tab? I was thinking about "Reopen last closed tab" but it might be confusing since after restoring the last closed tab, "last closed tab" it might be hard to understand what the last closed tab is. What are your thoughts?

Reply:
How about "Reopen recently closed tab"?

Reply:
Sounds good to me.